src/useIsStickyStuck/useIsStickyStuck.ts

```diff
@@ -0,0 +1,36 @@
+import { Ref, computed, onMounted, ref } from 'vue'
+import { MaybeRef } from '@/types/maybe'
+import { useIntersectionObserver } from '@/useIntersectionObserver'
+
+type UseIsStickyStuckOptions = {
+  rootMargin?: string,
+  boundingElement?: HTMLElement | Ref<HTMLElement | undefined>,
+}
+
+export function useIsStickyStuck(
+  element: HTMLElement | Ref<HTMLElement | undefined>,
```

Comment:
Trying to have all new compositions support getters. 
```suggestion
  element: MaybeRefOrGetter<HTMLElement | undefined>,
```